### ISSUE TITLE
Error while importing galpy.orbit

### ISSUE THREAD

**Issue by piiamt** [jobovy/galpy#426]
Hi,

I would like to start using galpy on windows 10 with anaconda (python version 3.7) and I have installed both GSL and galpy but whenever I want to simply import with the command "from galpy.orbit import Orbit", I get the following error

"
  File "<stdin>", line 1, in <module>
  File "C:\Users\PC\Documents\Anaconda3\lib\site-packages\galpy\orbit\__init__.py", line 1, in <module>
    from . import Orbits
  File "C:\Users\PC\Documents\Anaconda3\lib\site-packages\galpy\orbit\Orbits.py", line 53, in <module>
    _APY_GE_31= tuple(map(int,(astropy.__version__.split('.')))) > (3,0,5)
ValueError: invalid literal for int() with base 10: 'post1'
"

I assumed that there was something wrong with my installation but even after updating both GSL and galpy this error persists and I cannot integrate any orbits. When I import something else from galpy like potentials or action angles I do not get any errors.

(I have already installed and successfully used all the prerequisites such as numpy, scipy and matplotlib)

Has anyone had similar problems? I am quite stumped with this and would appreciate any help

**Comment by jobovy** [comment]
This is a problem that was fixed in #407 and the fix was released in the latest version v1.6, so it must be that you installed an older version (check with ``import galpy; print(galpy.__version__)``). Not sure why conda wouldn't have installed the latest version, you can try updating it using conda or use pip
```
pip install -U --no-deps galpy
```

**Comment by piiamt** [comment]
Yes, it turned out that anaconda was not really updating galpy from version 1.5.
It would seem that the conda update --all left many things unchanged and the order in which I updated packages separately was important - before updating the galpy prerequisites (numpy, scipy, matplotlib) I also had to update astropy separately. After updating first astropy and then the other packages one by one, galpy finally updated as well.
